CorsixTH/Lua/dialogs/staff_dialog.lua

```diff
@@ -260,7 +260,8 @@ function UIStaff:onMouseUp(button, x, y)
   end
   local repaint = Window.onMouseUp(self, button, x, y)
   -- Test for hit within the view circle
-  if button == "right" and is_in_view_circle(x, y, self.staff.profile.humanoid_class == "Handyman") then
+  if button == "right" and is_in_view_circle(x, y, self.staff.profile.humanoid_class == "Handyman")
+     or x > 18 and x < 170 and y > 19 and y < 42 then
```

Comment:
Could use an explanation. That's a lot of magic numbers.

Reply:
These are coordinates of name label

Reply:
Yes

Reply:
Or a table to label them instead (but could be overkill, and you'd change the inequalities to `>=` etc)
e.g. (change name to what is relevant)
```lua
local name_box = {
  x_min = 19
  x_max = 169
  y_min = 20
  y_max = 41
}
```

Though are these name box's coordinates not always specified in the main function `UIStaff:UIStaff` (when we draw we typically specify specific pixels or an `x+number` increment), rather than repeat them two times?

Reply:
> Or a table to label them instead (but could be overkill, and you'd change the inequalities to `>=` etc) e.g. (change name to what is relevant)
> 
> ```lua
> local name_box = {
>   x_min = 19
>   x_max = 169
>   y_min = 20
>   y_max = 41
> }
> ```
> 
> Though are these name box's coordinates not always specified in the main function `UIStaff:UIStaff` (when we draw we typically specify specific pixels or an `x+number` increment), rather than repeat them two times?

Could use the similar table to refactor machine menu later, because it does checking for the click same as my pull request